frontend/ui/widget/dictquicklookup.lua

```diff
@@ -245,11 +245,68 @@ function DictQuickLookup:update()
     -- Different sets of buttons if fullpage or not
     local buttons
     if self.is_fullpage then
-        -- Only a single wide close button, get a little more room for
-        -- closing by taping at bottom (on footer or on this button)
+        -- A save and a close button
         buttons = {
             {
                 {
+                    text = "Save as epub",
+                    callback = function()
+                        local InfoMessage = require("ui/widget/infomessage")
+                        local ConfirmBox = require("ui/widget/confirmbox")
+                        -- if forced_lang was specified, it may not be in our wiki_languages,
+                        -- but ReaderWikipedia will have put it in result.lang
+                        local lang = self.lang or self.wiki_languages_copy[1]
+                        -- Just to be safe (none of these chars, except ':' for uninteresting
+                        -- Portal: or File: wikipedia pages, should be in lookup_word)
+                        local cleaned_lookupword = self.lookupword:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','_')
```

Comment:
I think i may have cut and pasted it from there :) Guess I liked having that gsub there in case i needed more. But ok, I will use util.replaceInvalidChars at next rebase.